src/common.rs

```diff
@@ -39,8 +39,13 @@ pub(crate) fn normalize_anonymous_namespace(text: &str) -> String {
 }
 
 pub(crate) fn fix_symbol_name<'a>(name: &'a Name<'a>) -> Name<'a> {
-    static LLVM_NNN: Lazy<Regex> = Lazy::new(|| Regex::new(r"\.llvm\.[0-9]+$").unwrap());
-    let fixed = LLVM_NNN.replace(name.as_str(), "");
+    static COMPILER_NNN: Lazy<Regex> = Lazy::new(|| {
+        Regex::new(
+            r"((\.(cold|constprop|llvm|localalias|isra|part|str)(\.[0-9]+)?)|( ?\[clone.*\] ?))+$",
```

Comment:
I think `clone.*\]` will catch, for example, all the string `"clone a] [hello world]"`.
So if it isn't what you expect you should likely use something like `clone[^\]]*\]`

Reply:
Ah yes, good point. I'll write a test to cover it too.